scripts/deploy-ul-base.ts

```diff
@@ -0,0 +1,24 @@
+import { ethers } from "hardhat";
+
+async function main() {
+    const addrs = await ethers.getSigners();
+
+    console.log("Deploying contracts with the account:", addrs[0].address);
+    console.log("Account balance:", (await addrs[0].getBalance()).toString());
+
+    const ULR = await ethers.getContractFactory("UniversalLiquidatorRegistry");
```

Comment:
As additional step I'd like to set the `intermediateTokens` within this script. It's necessary setup. I think we can start with WETH, USDC, USDT.

Reply:
Fixed